stl/inc/format

```diff
@@ -2133,9 +2162,9 @@ _NODISCARD _OutputIt _Fmt_write(
 
         if (_Specs._Localized) {
             _Out = _Write_separated_integer(_Buffer_start, _Integral_end, _Groups,
-                _STD use_facet<numpunct<_CharT>>(_Locale).thousands_sep(), _Separators, _STD move(_Out));
+                _STD use_facet<numpunct<_CharT>>(_Locale._Get()).thousands_sep(), _Separators, _STD move(_Out));
             if (_Radix_point != _Result.ptr || _Append_decimal) {
-                *_Out++         = _STD use_facet<numpunct<_CharT>>(_Locale).decimal_point();
+                *_Out++         = _STD use_facet<numpunct<_CharT>>(_Locale._Get()).decimal_point();

```

Comment:
Pre-existing: It would be less verbose and more efficient to extract these `use_facet` calls. (There's a third one above, but this is conditional on `_Specs._Localized`, so unifying them all would be a bit of a headache.)